person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private_class_method def self.of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    if of_age? || @parent_permission
+      true
+    else
+      false
+    end
```

Comment:
- [**Optional**] Great job for using the correct logical operator issue raised by the previous reviewer :+1: I think there's no reason for explicitly returning true or false in this method. The method name itself(**can_use_services?**) return true by default else false if the statement is not true. what you have done in of_age? method can equally true in this method.